integration/harness_test.go

```diff
@@ -41,14 +41,18 @@ func infof(format string, args ...any) {
 var buildOnce sync.Once
 
 // run an integration test.
-func run(t *testing.T, actions ...action) {
+func run(t *testing.T, ftlConfigPath string, actions ...action) {
 	tmpDir := t.TempDir()
 
 	cwd, err := os.Getwd()
 	assert.NoError(t, err)
 
 	rootDir := internal.GitRoot("")
 
+	if ftlConfigPath != "" {
+		t.Setenv("FTL_CONFIG", filepath.Join(tmpDir, ftlConfigPath))
```

Comment:
Let's use a path into the `testdata` directory instead of one relative to the tmpDir. Otherwise we have a chicken and egg situation where the config can't be loaded until the module is copied over, and the config itself is used by FTL during startup.

Reply:
Also add a comment to `run` describing the above.

Reply:
ah good point, done on both counts